app/models/file.js

```diff
@@ -8,7 +8,7 @@ export default class File extends Model {
   @attr('string') extension;
   @attr('datetime') created;
 
-  @belongsTo('piece') piece;
+  @belongsTo('file', { inverse: null }) primarySource;
```

Comment:
Is er een expliciete reden om inverse op `null` te zetten? Anders zou ik de inverse relatie wel specifiëren

Reply:
Eigenlijk gewoon omdat de inverse relatie (nog) niet nodig is. Maar kan inderdaad gewoon toegevoegd worden.